impl/src/test/java/org/ehcache/GettingStarted.java

```diff
@@ -202,6 +211,21 @@ public void testStoreByValue() {
     cacheManager.close();
   }
 
+  @Test
+  public void testCacheEventListener() {
+    DefaultCacheEventListenerConfiguration cacheEventListenerConfiguration = CacheEventListenerBuilder
+        .newEventListenerConfig(ListenerObject.class, EventType.CREATED, EventType.UPDATED)
+        .unordered().asynchronous().build();
+    
+    final CacheManager manager = CacheManagerBuilder.newCacheManagerBuilder()
+        .withCache("foo",
+            CacheConfigurationBuilder.newCacheConfigurationBuilder()
+                .addServiceConfig(cacheEventListenerConfiguration)
+                .buildConfig(Object.class, Object.class)).build(true);
+
```

Comment:
You should probably add a few puts and removes to demonstrate that CREATED and UPDATED but not REMOVED events are getting fired.